app/scripts/common-transforms.js

```diff
@@ -35,7 +55,48 @@ var commonTransforms = (function(_) {
             });
             return buckets;
         },
+        /**
+        * Gives two level heirarchies for the aggregrations - and the thir hierarchy as just an info
+        * Example filter level1 by date and level2 by city and pulblisher as info
+           {
+                date:[{
+                    date: 1455657767
+                    city:{
+                        city: "Los Angeles",
+                        info: '"abc.com", "rg.com"''
+                    }
+                },
+    
+           }
+        */
+        infoBuckets: function(buckets,level1,level2,info){
 
+            buckets = _.reduce(buckets, function (results, bucket) {
+                var obj_level1 = {};
```

Comment:
Please follow the project's existing variable naming conventions and use camelCase names instead of underscored names.